ios/RPCModule.m

```diff
@@ -28,7 +28,7 @@ @implementation RPCModule
   NSString *documentsDirectory = [paths objectAtIndex:0];
 
   // Write to user's documents app directory
-  NSString *fileName = [NSString stringWithFormat:@"%@/zecwallet-lite.dat.txt",
+  NSString *fileName = [NSString stringWithFormat:@"%@/wallet.dat.txt",
```

Comment:
so then this file needs to be renamed too? Everything that I didn't change needed a filename change as well, as far as I tried ..

Reply:
ah on second thought, I think maybe this is the file that the program is going to create maybe - not one that is checked in so it's fine.

Reply:
To match `zingolib` this should be `zingo-wallet.dat.txt`

Reply:
Sounds good to me, I'll do it right away.

Reply:
done! I hope the `-` don't break something in the ios env.